ocs_ci/krkn_chaos/aggressive_clone_operations.py

```
@@ -0,0 +1,1001 @@
+                workload.stop_workload()
+                workload.cleanup_workload()
+
+        with self._pool_lock:
+            entries = list(self._clone_pool)
+        for entry in entries:
+            self._delete_tracked_clone(entry)
+
+        self._cleanup_orphan_aggressive_resources()
+        self._verify_pool_reclamation()
+
+    def _cleanup_orphan_aggressive_resources(self):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_cleanup_orphan_aggressive_resources`

Add return type annotation: `None`

(ANN202)